easybuild/easyblocks/m/motioncor2.py

```diff
@@ -140,5 +154,9 @@ def sanity_check_step(self):
             'files': [os.path.join('bin', x) for x in ['motioncor2', self.motioncor2_bin]],
             'dirs': []
         }
-
-        super(EB_MotionCor2, self).sanity_check_step(custom_paths)
+        for f in custom_paths['files']:
+            fp = os.path.join(self.installdir, f)
+            if not os.path.isfile(fp):
+                raise EasyBuildError("File not installed {}".format(fp))
+        if not self.cfg.get('skip_rpath_check', False):
+            super(EB_MotionCor2, self).sanity_check_step(custom_paths)
```

Comment:
There's an easier way to only skip the RPATH sanity check:

```python
def sanity_check_rpath(self, *args, **kwargs):
    """Custom implementation of RPATH sanity check: allow skipping via skip_rpath_sanity_check."""
    if self.cfg.get('skip_rpath_sanity_check', False):
        self.log.info("Skipping RPATH sanity check, as specified via 'skip_rpath_sanity_check'...")
    else:
        super(EB_MotionCor2, self).sanity_check_rpath(*args, **kwargs)
```

Then you don't have to partially replicate the sanity check for files yourself either...

And maybe we should make `skip_rpath_sanity_check` a general easyconfig parameter in framework instead?